inst/qml/common/classical/BarPlot.qml

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (C) 2013-2022 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+import QtQuick			2.12
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP				1.0
+
+Section
```

Comment:
can' this file be the same as the Bayesian one? you could add a boolean property to switch between `"Confidence interval"` and `"Credible interval"`.

Reply:
@foger3 Here is how to do it https://github.com/Kucharssim/jaspAnova/commit/a0dfd9d6304ed48e270722e6f98b914b82d34281 (it was easier to do it than to explain here how to do it, you should be able to cherry-pick this commit to include it into this PR).

I also renamed the options in the commit to follow the option names style guide (https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/guide-option-names.md). Please follow this guide in the future as well, otherwise we will end up renaming the options anyways very soon - but it's better to just get it right from the start

Reply:
Hi. Didnt see your comment and commited my own version before that. It considers the switch between confidence and credible interval in the display and also place the BarPlots.qml in the common section, seems to work. Yes, the naming got messed up, didnt realize it until late:/